services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/qrcode/QrCodeTemplateLoader.java

```diff
@@ -1,42 +1,46 @@
 package app.coronawarn.server.services.distribution.assembly.qrcode;
 
+import static org.springframework.core.io.ResourceLoader.CLASSPATH_URL_PREFIX;
+
 import app.coronawarn.server.services.distribution.config.DistributionServiceConfig;
 import com.google.protobuf.ByteString;
-import java.io.File;
-import java.io.FileInputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ResourceLoader;
 import org.springframework.stereotype.Component;
 
 @Component
 public class QrCodeTemplateLoader {
 
   private static final Logger logger = LoggerFactory.getLogger(QrCodeTemplateLoader.class);
 
-  private final DistributionServiceConfig config;
+  @Autowired
+  DistributionServiceConfig config;
```

Comment:
These changes are from me and shoulnd't have creeped in here. I'll remove them for a separate PR...